src/test/java/hudson/plugins/sauce_ondemand/SauceOnDemandBuildActionTest.java

```diff
@@ -85,6 +89,30 @@ public void testGetSauceBuildAction () throws Exception {
         assertEquals("fakeuser", sauceBuildAction.getCredentials().getUsername());
     }
 
+    /**
+     * Verifies get sauce action for maven build
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testGetSauceBuildActionMavenBuild() throws Exception {
+        hudson.maven.MavenModuleSet project = jenkins.createMavenProject();
+        TestSauceOnDemandBuildWrapper bw = new TestSauceOnDemandBuildWrapper(
+            SauceCredentials.migrateToCredentials("fakeuser", "fakekey", "unittest")
+        );
+        project.getBuildWrappersList().add(bw);
+        project.setScm(new SingleFileSCM("pom.xml",getClass().getResource("/pom.xml")));
+        project.setGoals("clean");
+        MavenModuleSetBuild build =  project.scheduleBuild2(0).get(1, TimeUnit.MINUTES);
+        SauceOnDemandBuildAction buildAction = new SauceOnDemandBuildAction(build, bw.getCredentialId());
+        build.addAction(buildAction);
+        final MavenBuild mavenBuildMock = mock(MavenBuild.class);
+        when(mavenBuildMock.getParentBuild()).thenReturn(build);
+        SauceOnDemandBuildAction  sauceBuildAction= SauceOnDemandBuildAction.getSauceBuildAction(mavenBuildMock);
```

Comment:
nitpicky, but do you want to fix the spacing?
you have 2 before sauceBuildAction, and 0 after it